src/services/legal-services.ts

```diff
@@ -168,13 +169,23 @@ export default class LegalServices {
   }
 
   /**
-   * Creates (posts) a draft (temporary) business record.
-   * Must be logged in to use this.
-   * Throws an exception on error.
+   * Creates (posts) a draft business record, which is used to bootstrap a new business.
+   * @param businessRequest the object body of the request
+   * @returns the filing object associated with the temporary business
    */
-  static async createBusiness (businessRequest: any): Promise<any> {
+  static async createDraftBusiness (businessRequest: any): Promise<any> {
     const url = `businesses?draft=true`
+
     return axios.post(url, businessRequest)
+      .then(response => {
+        const filing = response?.data?.filing
+        if (!filing) {
+          // eslint-disable-next-line no-console
+          console.log('createDraftBusiness() error - invalid response =', response)
+          throw new Error('Invalid filing')
+        }
+        return filing
```

Comment:
This does 2 things:
- checks that the response contains the expected data
- returns the filing (instead of the whole response)